### DISCUSSION TITLE
The image extracted has size greater than the pdf page size [Reopening] #458

### DISCUSSION THREAD

**Discussion by ZafarShadman09** [pymupdf/PyMuPDF#1350] [Q&A] [answered]
Image extracted from the PDF have greater resolution than the pdf resolution.

How to get the original dimensions of the image present inside the PDF?

**Comment by JorjMcKie** [comment, marked as answer]
First of all: this is a typical discussion item, not an issue.

Second: have you read my today's comment on the original issue?

Third, as per topic "resolution": there is no contradiction, because a PDF has nothing like that. You can make a page pixmap at some desired resolution, but this is an entirely different thing.
